src/common/modules/UnicodeFontHandler.js

```diff
@@ -60,7 +55,7 @@ function changeFont(atext, afont) {
  * @returns {string}
  */
 function capitalizeEachWord(atext) {
-	// Requires Firefox/Thunderbird 78
+	// Regular expression Unicode property escapes and lookbehind assertions require Firefox/Thunderbird 78
```

Comment:
Wow okay, that's interesting. I can confirm that and would suggest to add a link here:
```suggestion
	// Regular expression Unicode property escapes and lookbehind assertions require Firefox/Thunderbird 78, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#bcd:javascript.builtins.RegExp
```

And there is no way to work around this?
Checking [the release calendar](https://wiki.mozilla.org/Release_Management/Calendar), I however can say we are quite lucky that Firefox 78 was an ESR release. The previous ESR release stopped being supported at 2020-09-22 https://en.wikipedia.org/wiki/Firefox_version_history#Firefox_78_through_87

So I guess it's fine technically, but interesting.

Did not re-check now, but if that also applies to Awesome Emoji Picker, I'd be more reluctant to change it there. Because this is a new add-on, so we thus cannot disappoint users, who may be stuck on legacy versions or so.

Reply:
OK, will do.

No, Unicode property escapes are the only way for this function to properly support all languages/locales, as the JS character classes [only support English characters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes#Types). Also see [this article](https://hacks.mozilla.org/2020/06/new-in-firefox-78/) about new features in Firefox 78.

No, this is for the Unicode font conversion feature and does not apply to the Awesome Emoji Picker. As you probably know, Thunderbird only releases ESR versions, so both add-ons depend on WebExtension/MailExtension APIs that were added to Thunderbird 78 and thus will require Thunderbird 78, but that does not effect Firefox.